CommunityFridgeMapApi/functions/dev/get_all/app.py

```diff
@@ -0,0 +1,54 @@
+import os
+import logging
+from botocore.exceptions import ClientError
+import json
+try:
+    from db import get_ddb_connection
+except:
+    #For Unit Testing Purposes
+    import sys
```

Comment:
the one thing about lambda layers is that it makes it a bit harder to unit test your code. So there's a tradeoff there